### ISSUE TITLE
True zen recognizes the relativity in all things

### ISSUE THREAD

**Issue by KurtPreston** [zengabor/zenscroll#42]
Hey, great package!  Recently noticed an issue where zenscroll will not successfully scroll to any element contained in a `position: relative` div.

Demo here:
http://requirebin.com/?gist=ae3708359c2ad71651d03fb9bdfc9115

In it, you will see three buttons.  The second one will not work because div `b` is contained inside a div with `position: relative`, and will instead scroll to the top of the container div.

**Comment by zengabor** [comment]
Thank you very much for reporting it, especially that you prepared an example. I am looking into it.

Great subject by the way! :)

**Comment by pitAlex** [comment]
Its because the library is made to use elem.offsetTop. But you should change it to walk up the given element's parents until it reaches the zenscroll container.
Is the fix planned any time soon?

**Comment by TiagoPortfolio** [comment]
![One Eternity Later](https://i.ytimg.com/vi/U7CZcd-UYmU/maxresdefault.jpg "One Eternity Later")

**Comment by zengabor** [comment]
You are right, and sorry about that. I was hoping that eventually all major browsers implement smooth scrolling, which is preferred over a custom JavaScript solution. Fortunately, this [has happened](https://caniuse.com/css-scroll-behavior) since then.
